xml/signals/FS-1987/index.shtml

```diff
@@ -4,7 +4,7 @@
   <meta name="generator" content="HTML Tidy for HTML5 for Apple macOS version 5.8.0">
   <!-- Copyright ($Author$) 2009 -->
 
-  <title>JMRI: FS - Italian State Railways - Signaling Definition</title>
+  <title>JMRI: AAR-1946 Signaling Definition</title>
```

Comment:
Is this change intended? `FS - Italian State Railways` --> `AAR-1946`?

Reply:
Probably not!  Looks like a cut and paste error, now fixed.  Thanks for catching it.